src/MissionManager/ComplexMissionItem.h

```diff
@@ -15,6 +15,10 @@
 
 #include <QSettings>
 
+#include <QGCToolbox.h>
```

Comment:
@dakejahl  I noticed this after I merged. QGC includes should be "foo.h", only Qt includes are "<foo>". Also you should keep the Qt and QGC include blocks together. So in this case the include you added should have gone right underneath the other existing QGC include at the top of the file without an extra newlines. Make sense?

I've fixed it up in my terrain pull which merged conflicted in these file which is how I notied.

Reply:
Yup makes sense, I was not aware of the convention,